c/json.c

```diff
@@ -1368,10 +1477,15 @@ JsonToken *jsonNextToken(JsonParser* parser) {
   lookahead = jsonTokenizerLookahead(tokenizer);
   tokenizer->lastTokenLineNumber = tokenizer->lineNumber;
   tokenizer->lastTokenColumnNumber = tokenizer->columnNumber;
+  /* printf("JOE jsonNextToken lookahead=0x%x '%c'\n",(int)lookahead,(char)lookahead);
```

Comment:
Dead code.

Reply:
leaving in until converted to debug traces.